### DISCUSSION TITLE
How do i host it 24/7?

### DISCUSSION THREAD

**Discussion by Nikakere** [jagrosh/MusicBot#992] [Questions]
is there any way to host the bot 24/7 for free? f.e would it be possible to host it on heroku?

**Comment by MichailiK** [comment]
We advise against [using free hosts](https://jmusicbot.com/hosting/) due to the poor quality you may experience. Yes, it's possible, but it can be an awful experience. Additionally, Heroku isn't meanr for hosting discord bots anyway.
